test/testinput/3densvar.yaml

```diff
@@ -79,6 +79,9 @@ cost function:
       obserrors:
         type: percent
         value: 30
+    obs filters:
```

Comment:
should we keep a test with filters and one not or the 4densvar is acting as a non filtered test ?

Reply:
New test included